CHANGELOG.rst

```diff
@@ -2,6 +2,21 @@
 Changelog
 =========
 
+0.17.0 (YYYY-MM-DD)
+-------------------
+
+* Now requires ``web-poet >= 0.15.1``.
+
+* Add :class:`~scrapy_poet.page_input_providers.HttpRequestProvider`.
```

Comment:
Should we explain in more user-facing terms, e.g. "HttpRequest dependency is now supported, via HttpRequestProvider (enabled by default)"